Gemfile

```diff
@@ -45,7 +45,7 @@ gem 'uglifier', '>= 1.3.0'
 
 # Use jquery as the JavaScript library
 gem 'jquery-rails', '~> 4.0.4'
-
+gem 'listen', '~> 3.1', '>= 3.1.3'
```

Comment:
Why are you adding this gem?  Guard already includes it for us.  (See gemfile.lock)